### PR TITLE
add general debugging guide

### PR DESCRIPTION
**Pull request by davemooreuws**
Includes quick video for VSCode debugging and general overview and instructions.

### CODE REVIEW

**Comment by tjholm** [comment]
@davemooreuws do you think its worth clarifying that this is for js/ts only?

**Comment by davemooreuws** [comment]
> @davemooreuws do you think its worth clarifying that this is for js/ts only? We may add debugging guides for additional languages in future, and they will probably be distinct enough to warrent their own guides.

Edited

File: docs/getting-started.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ Access your [local development dashboard](./local-dashboard.md) now and start st
 
 Your development dashboard is a powerful tool designed to help you move faster when developing new features. With the API explorer, you can easily access automatically generated endpoints and path params. Additionally, the Schedules Explorer allows you to trigger your schedules with ease, making it easier than ever to stay on top of your tasks. Make the most of your local development process with this essential tool.
 
-Watch this brief video to see the development dashboard in action. You'll get a firsthand look at how the dashboard streamlines your development process by showcasing the debugging APIs and schedules in the popular code editor, VS Code.
+Watch this brief video to see the development dashboard in action. You'll get a firsthand look at how the dashboard streamlines your development process by showcasing the [debugging](./guides/debugging.md) APIs and schedules in the popular code editor, VS Code.
```
Comment (HomelessDinosaur) [inline docs/getting-started.md:183]:
```suggestion
Watch this brief video to see the development dashboard in action. You'll get a firsthand look at how the dashboard streamlines your development process by showcasing the [debugging](./guides/debugging.md) of APIs and schedules in the popular code editor, VS Code.
```

File: docs/guides/debugging.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+---
+title: Debugging with Nitric
+description: Experience real-time updates, API exploration, and schedule debugging in Nitric's local development dashboard.
+---
+
+Debugging is a critical skill for any developer, as it allows them to identify and fix errors in their code. Without debugging, developers would be unable to troubleshoot issues and improve the performance and functionality of their applications.
+
+Fortunately, Nitric provides a powerful solution for debugging that can be used with a wide range of programming languages and code editors. By leveraging the Nitric local dashboard, developers can quickly and easily identify issues in their code and use their favorite language debugger and code editor to make the necessary fixes. This streamlined process can save developers time and frustration, allowing them to focus on building high-quality applications that meet the needs of their users.
+
+Here's a quick video demonstrating how to use Nitric's local dashboard for debugging with VSCode:
```
Comment (HomelessDinosaur) [inline docs/guides/debugging.md:10]:
```suggestion
Here's a quick video demonstrating how to use Nitric's local dashboard for debugging with VS Code:
```

**Review by HomelessDinosaur — APPROVED** [review]

**Pull request merged**